### PR TITLE
Release finschia-sdk v0.48.0

### PR DESCRIPTION
**Pull request by zemyblue**
# Release v0.48.0

### Features
* (x/auth) #1011
* (server/grpc) #1017 
* (x/auth) #1085
* (proto) #1087

### Improvements
* (third_party/proto) #1037
* (ostracon) #1057
* (x/foundation) #1072
* (cli) #1086
* (ostracon) #1089
* (ostracon) #1099
* (refactor) #1114
* (x/token) #1128
* (x/token) #1129
* (x/token) #1130
* (x/collection) #1131
* (x/collection) #1133
* (x/token) #1137
* (x/collection) #1139
* (x/token) #1140
* (ostracon) #1142

### Bug Fixes
* (ledger) #1040
* (x/foundation) #1053
* (baseapp) #1091
* (x/foundation) #1108

### State Machine Breaking
* (x/foundation) #999
* (x/collection) #1102
* (x/collection) #1105
* (x/collection) #1106

### Event Breaking Changes
* (refactor) #1090
* (x/bank) #1093
* (baseapp) #1092
* (x/foundation) #1053

### Build, CI
* (build,ci) #1043

### Document Updates
* (readme) #997
* (docs) #1094

### CODE REVIEW

**Review by jaeseung-bae — DISMISSED** [review]
LGTM

File: RELEASE_CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,58 @@
...
+* (x/collection) [\#1131](https://github.com/Finschia/finschia-sdk/pull/1131) add additional unittest of x/collection(`MsgIssueFT`, `MsgMintFT`, `MsgBurnFT`)
+* (x/collection) [\#1133](https://github.com/Finschia/finschia-sdk/pull/1133) Refactor unittest for `SendFT`, `OperatorSendFT`, `AuthorizeOperator`, and `RevokeOperator` to check more states
+* (x/token) [\#1137](https://github.com/Finschia/finschia-sdk/pull/1137) Add test for event compatibility
+* (x/collection) [\#1139](https://github.com/Finschia/finschia-sdk/pull/1139) refactor overall unittests of `x/collection`
+* (x/token) [\#1140](https://github.com/Finschia/finschia-sdk/pull/1140) Refactor unittest for `x/token`
+* (ostracon) [\#1142](https://github.com/Finschia/finschia-sdk/pull/1142) Bump up ostracon from v1.1.2-0.20230822110903-449aa3148b12 to v1.1.2
+
+### Bug Fixes
+* (ledger) [\#1040](https://github.com/Finschia/finschia-sdk/pull/1040) Fix a bug(unable to connect nano S plus ledger on ubuntu)
+* (x/foundation) [\#1053](https://github.com/Finschia/finschia-sdk/pull/1053) Make x/foundation MsgExec propagate events
```
Comment (0Tech) [inline RELEASE_CHANGELOG.md:31]: It's event breaking change.

We also have to check other messages, because new stanza has been introduced.

**Review by 0Tech — CHANGES_REQUESTED** [review]

File: RELEASE_CHANGELOG.md (hunk 1 above)
Comment (zemyblue) [reply RELEASE_CHANGELOG.md:31]: @0Tech , Do you mean 1040 and 1053 are also need to be added in event breaking change section?

Comment (0Tech) [reply RELEASE_CHANGELOG.md:31]: I've checked other messages, it seems to be OK.

Comment (0Tech) [reply RELEASE_CHANGELOG.md:31]: > @0Tech , Do you mean 1040 and 1053 are also need to be added in event breaking change section?

Just for 1053.

Comment (zemyblue) [reply RELEASE_CHANGELOG.md:31]: Ok, I'll add copy 1053 and more add it to event breaking change section.

**Review by 0Tech — APPROVED** [review]

**Review by jaeseung-bae — APPROVED** [review]

**Pull request merged**